pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -643,7 +647,41 @@ func podAffinity(i *infinispanv1.Infinispan, matchLabels map[string]string) *cor
 	return i.Spec.Affinity
 }
 
-// deploymentForInfinispan returns an infinispan Deployment object
+func (r *ReconcileInfinispan) updatePodsLabels(m *infinispanv1.Infinispan, podList *corev1.PodList) error {
+	if len(podList.Items) == 0 {
+		return nil
+	}
+
+	labelsForPod := PodLabels(m.Name)
+	m.AddOperatorLabelsForPods(labelsForPod)
+	m.AddLabelsForPods(labelsForPod)
```

Comment:
@rigazilla Is there any reason we can't add these calls inside `PodLabels` so that it doesn't have to be repeated in multiple places?

Reply:
I wanted to leave the possibility to update only CR or operator labels, I'm not sure if that case will ever happens anyway